taxonomies/ingredients.txt

```diff
@@ -3,6 +3,7 @@
 synonyms:en:pasteurised, pasteurized
 synonyms:es:zumo, jugo
 synonyms:es:sirope,jarabe
+synonyms:fr:jus,jugo
```

Comment:
jugo is not a French word

Reply:
I'm sure I came across an entry - I'll remove it.